### PR TITLE
Feature/query pane redesign tab

### PR DESCRIPTION
**Pull request by ram-primathon**
## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by hetunandu — CHANGES_REQUESTED** [review]
Also create this for the response section of the query

**Review by hetunandu — CHANGES_REQUESTED** [review]

File: app/client/src/pages/Editor/QueryEditor/Form.tsx [hunk 1]
```diff
@@ -115,28 +108,38 @@ const TabContainerView = styled.div`
...
 const SettingsWrapper = styled.div`
   padding: 5px 23px;
 `;
 
-const AddWidgetButton = styled(BaseButton)`
-  &&&& {
-    height: 36px;
-    max-width: 125px;
-    border: 1px solid ${Colors.GEYSER_LIGHT};
+const AddWidgetButton = styled.a`
```
[the remark this answers is not in the document]
Comment (ram-primathon) [reply app/client/src/pages/Editor/QueryEditor/Form.tsx:119]: done

File: app/client/src/pages/Editor/QueryEditor/Form.tsx [hunk 2]
```diff
@@ -397,26 +421,36 @@ const QueryEditorForm: React.FC<Props> = (props: Props) => {
...
+                                {isTableResponse ? (
+                                  <Table data={output} />
+                                ) : (
+                                  <JSONViewer src={output} />
+                                )}
+                              </OutputTableWrapper>
+                            ),
+                          },
+                        ]}
+                      />
```
[the remark this answers is not in the document]
Comment (ram-primathon) [reply app/client/src/pages/Editor/QueryEditor/Form.tsx]: done

File: app/client/src/pages/Editor/QueryEditor/Form.tsx [hunk 3]
```diff
@@ -397,26 +421,36 @@ const QueryEditorForm: React.FC<Props> = (props: Props) => {
...
-                              onClick={onAddWidget}
-                            />
-                          </Boxed>
-                        )}
-                      </OutputHeader>
-                      {isTableResponse ? (
-                        <Table data={output} />
-                      ) : (
-                        <JSONViewer src={output} />
+                    <OutputTabWrapper>
```
[the remark this answers is not in the document]
Comment (ram-primathon) [reply app/client/src/pages/Editor/QueryEditor/Form.tsx]: done

File: app/client/src/pages/Editor/QueryEditor/Form.tsx [hunk 4]
```diff
@@ -146,11 +149,34 @@ const FieldWrapper = styled.div`
...
 `;
 
 const OutputWrapper = styled.div``;
 
+const OutputTabWrapper = styled.div`
+  position: relative;
+
+  &&&& ul.react-tabs__tab-list {
+    padding-left: 0;
+    border: none;
```
[the remark this answers is not in the document]
Comment (ram-primathon) [reply app/client/src/pages/Editor/QueryEditor/Form.tsx]: removed

File: app/client/src/pages/Editor/QueryEditor/Form.tsx [hunk 5]
```diff
@@ -146,11 +149,34 @@ const FieldWrapper = styled.div`
...
+    margin-top: 0;
+  }
+
+  &:hover {
+    color: #484848;
+    text-decoration: none;
+  }
 `;
 
 const OutputWrapper = styled.div``;
```
[the remark this answers is not in the document]
Comment (ram-primathon) [reply app/client/src/pages/Editor/QueryEditor/Form.tsx]: done

**Review by hetunandu — APPROVED** [review]

**Pull request merged**